gen/src/DepoTransform.cxx

```diff
@@ -145,32 +157,47 @@ WireCell::Configuration Gen::DepoTransform::default_configuration() const
     // type-name for the DFT to use
     cfg["dft"] = "FftwDFT";
 
+    cfg["process_planes"] = Json::arrayValue;
+
     return cfg;
 }
 
 bool Gen::DepoTransform::operator()(const input_pointer& in, output_pointer& out)
 {
+
     if (!in) {
         log->debug("EOS at call={}", m_count);
         ++m_count;
         out = nullptr;
         return true;
     }
 
+
     auto depos = in->depos();
     size_t ndepos_used=0;
 
     Binning tbins(m_readout_time / m_tick, m_start_time, m_start_time + m_readout_time);
     ITrace::vector traces;
+
     for (auto face : m_anode->faces()) {
         // Select the depos which are in this face's sensitive volume
         IDepo::vector face_depos = Aux::sensitive(*depos, face);
-        ndepos_used += face_depos.size();
+	ndepos_used += face_depos.size();       
 
         int iplane = -1;
         for (auto plane : face->planes()) {
             ++iplane;
 
+	    int plane_index = plane->planeid().index();
+	    
+	    log->debug("Checking: Plane Testing {} and looking For {}", plane_index, m_process_planes[0]);
+	   
+	    if (std::find(m_process_planes.begin(),  m_process_planes.end(), plane_index) == m_process_planes.end()) {   	      
+	      continue;
+	    }
+	    
+	    log->debug("Working on: Plane Testing {} and looking For {}", plane_index, m_process_planes[0]);
```

Comment:
These two logs will be rather verbose and not provide any useful novel information.  They should be removed.